coda/templates/mdstore/node.html

```diff
@@ -31,6 +31,10 @@ <h3 class="main">
             <th>Full</th>
             <td> <i class="icon-dashboard"></i> {{ filled }}</td>
         </tr>
+        <tr>
+            <th>Status</th>
+            <td> <i class="icon-dashboard"></i> {{ node.get_status_display }}</td>

```

Comment:
For this and the other modified template, I think we should change the icon used to be something like maybe a power button or something along those lines.

Reply:
I personally like the move to the green/red highlights, but we should see if others like it or would prefer to stick with icons for consistency.

Reply:
thoughts @ldko?

Reply:
We have similar setup for Premis events. I just used the same idea. :)

Reply:
I think the red and green text background is fine.